### ISSUE TITLE
Analyze stage fails when repo doesn't exist on container registry

### ISSUE THREAD

**Issue by ShadowJonathan** [pivotal/kpack#415]
I fired up a new cluster with `kpack` and `harbor` installed, harbor doesn't display or acknowledge that repos exist until an image is pushed to them, this results in this error in the analyze stage;
(domain, repo and image names changed for privacy)
```
ERROR: failed to get previous image: connect to repo store 'HARBOR.DOMAIN/REPO/IMAGE': GET https://HARBOR.DOMAIN/v2/REPO/IMAGE/manifests/latest: NOT_FOUND: repository REPO/IMAGE not found
```

To reproduce:
1. deploy kpack 0.0.9
2. deploy harbor
3. create a project and a corresponding robot account on harbor
4. create a clusterbuilder with image `heroku/buildpacks:18`
5. add robot account credentials, roll up into service account
6. apply image for application, watch build

**Comment by matthewmcnew** [comment]
Hi @ShadowJonathan, I believe this is an [known lifecycle issue](https://github.com/buildpacks/lifecycle/issues/303) that has been fixed with [lifecycle release 0.8.0](https://github.com/buildpacks/lifecycle/releases/tag/v0.8.0). 

The heroku team will need to upgrade their builder to use lifecycle 0.8.0 for this to be fixed. In the meantime you can use CustomBuilders with the heroku buildpacks or try the paketo builders.

**Comment by matthewmcnew** [comment]
I am going to close this issue. Feel free to reopen if needed.
